tests/cases/compiler/constEnumErrors.ts

```diff
@@ -9,10 +9,10 @@ module E {
 const enum E1 {
     // illegal case
     // forward reference to the element of the same enum
-    X = Y, 
+    X = Y, // error

```

Comment:
Additionally, in general I'm not sure how I feel about these. They're great when initially working on tests, but in the event that we decide something is no longer an error, they tend to stick around and become confusing.